### PR TITLE
ooniprobe 3.6.0

### PR DESCRIPTION
**Pull request by chenrui333**
[`action-homebrew-bump-formula`](https://github.com/dawidd6/action-homebrew-bump-formula)

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by carlocab** [comment]
```
internal/engine/resourcesmanager/resourcesmanager.go:47:12: pattern *.mmdb.gz: no matching files found
```

File: Formula/ooniprobe.rb [hunk 1]
```diff
@@ -15,6 +15,9 @@ class Ooniprobe < Formula
   depends_on "go" => :build
 
   def install
+    # fetch resources
+    system "go", "run", "./internal/cmd/getresources"
```
Comment (SMillerDev) [inline Formula/ooniprobe.rb:19]: Is this downloading unversioned unchecksummed things?

Comment (chenrui333) [reply Formula/ooniprobe.rb:19]: oh yes, maybe I can install the resource. Let me give that a try.

**Comment by chenrui333** [comment]
Will reopen this later.

File: Formula/ooniprobe.rb (hunk 1 above)
Comment (bassosimone) [reply Formula/ooniprobe.rb:19]: The download is both versioned and checksummed. The code that implements it is at https://github.com/ooni/probe-cli/blob/master/internal/cmd/getresources/getresources.go#L46 and the information about what to download is at https://github.com/ooni/probe-cli/blob/master/internal/engine/resources/assets.go. (Thank you @SMillerDev and @chenrui333!)

**Pull request closed without merging**